packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss.vue

```diff
@@ -236,6 +236,8 @@ const modelConfig = ref<ModelConfiguration>();
 
 const modelConfigId = computed<string | undefined>(() => props.node.inputs[0]?.value?.[0]);
 const datasetId = computed<string | undefined>(() => props.node.inputs[1]?.value?.[0]);
+const policyInterventions = computed(() => props.node.inputs[2].value);
```

Comment:
Would be nicer if naming is a bit more consistent, you seem to be inter-swapping`policyId` and `polcyInterventions`, and `policyInterventionId`.

Reply:
Shouldn't be `policyInterventionIds` and not `policyInterventionsId`. Just to be pedantic?

Reply:
I will rename all to policyInterventionId - no plural anywhere
1 ID is multiple interventions but one policy intervention